pydm/widgets/spinbox.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, parent=None, init_channel=None):
         self.setDecimals(0)
         self.app = QApplication.instance()
 
+        self.setAccelerated(True)
+
     def event(self, event):
```

Comment:
@hhslepicka, the behavior of the `Key_Up` and `Key_Down` treated in this method differs from clicking with the mouse in the arrows of the widget. The way it is handled here the value is sent to the channel when the key is pressed. Is this the behavior you guys want? Also, what do you think of using the `Key_Enter` in addition to the `Key_Return` to trigger the `send_value` method?
If you want I could add these changes to this pull request too.

Reply:
Hi @fernandohds564,
- **Yes** for the Key_Up and Key_Down to become similar to the button press
- **Yes** for adding the Key_Enter in addition to the Key_Return to trigger `send_value`

Thank you for adding the `setAccelerated` line!